query/src/main/java/io/keen/client/java/result/EXAMPLES.md

```diff
@@ -0,0 +1,14 @@
+Example of processing the results of a group_by query:
```

Comment:
Nit: You can delete this file (it's a leftover from my PR outlining the inheritance-based query result approach, IIRC).